### ISSUE TITLE
Issue about my own data

### ISSUE THREAD

**Issue by HeSongy** [xuxw98/ESAM#57]
Thank you for your work!

I try `CUDA_VISIBLE_DEVICES=0 python vis_demo/stream_demo.py --data_root ./data/data` in my own data(Replica room0)
but It seems something wrong when model Inference:
```
            with torch.no_grad():
                result = self.model.test_step(collate_data)
```
for the result of each frame : 

![Image](https://github.com/user-attachments/assets/fbac8460-845a-4179-9b8b-a8472ee3a657)

**It seems model pred nothing.**

----------------------------------------------

I use `--config configs/ESAM-E_CA/ESAM-E_online_stream.py` and `--checkpoint ESAM-E_CA_online_epoch_128.pth`

**Comment by HeSongy** [comment]
Problem solved
just Change the `inst_score_thr` in config file `ESAM-E_online_stream.py`

**Comment by xuxw98** [comment]
Sounds great! It seems that all masks are filtered out due to a large threshold.

**Comment by HeSongy** [comment]
> Sounds great! It seems that all masks are filtered out due to a large threshold.

other problem in #53 about `stream_demo.py`, do you meet the question before. Thanks

**Comment by xuxw98** [comment]
I think maybe you can adjust the hyperparameters on instance Merging to see whether it is due to a large domain gap between ScanNet and your own data.

**Comment by Griseo-Kaslana** [comment]
> I think maybe you can adjust the hyperparameters on instance Merging to see whether it is due to a large domain gap between ScanNet and your own data.

Yeah! You are absolutely right. I have added a clarification in my issue(https://github.com/xuxw98/ESAM/issues/53#issuecomment-2827740187). It only requires adding the following line of code.
`depth = depth / 6.5535`

**Comment by HeSongy** [comment]
Problem solved! Thanks very much!
